frontend/src/v5/ui/routes/viewer/groups/groupItem/groupItem.styles.ts

```diff
@@ -0,0 +1,138 @@
+/**
+ *  Copyright (C) 2022 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+import Checkers from '@assets/images/checkers.svg';
+import { ComponentToString } from '@/v5/helpers/react.helper';
+import { StyledIconButton } from '@/v4/routes/teamspaces/components/tooltipButton/tooltipButton.styles';
+import styled, { css } from 'styled-components';
+import { isV5 } from '@/v4/helpers/isV5';
+
+export const GroupsTreeListItem = styled.li`
+	cursor: default;
+	position: relative;
+`;
+
+const GroupsTreeListItemContainerV4 = css<{$highlighted?: boolean}>`
+	background-color: ${({ $highlighted }) => ($highlighted ? '#F7F7F7' : '#FFFFFF')};
+	border-bottom: 1px solid #DCDCDC;
+`;
+
+const GroupsTreeListItemContainerV5 = css<{$highlighted?: boolean}>`
+	background-color: ${({ $highlighted, theme: { palette } }) => ($highlighted ? palette.base.lightest : palette.primary.contrast)};
+	border-bottom: 1px solid  ${({ theme: { palette } }) => palette.base.lightest};
+`;
+
+export const GroupsTreeListItemContainer = styled.div<{$highlighted?: boolean, $depth }>`
+	padding-left: ${({ $depth }) => $depth * 10}px;
+	min-height: 41px;
+	align-items: center;
+	display: flex;
+	${() => (isV5() ? GroupsTreeListItemContainerV5 : GroupsTreeListItemContainerV4)}
+`;
+
+const IconSize = css`
+	width: 30px;
+	height: 28px;
+`;
+
+const PseudoElement = css`
+	${IconSize}
+	content: '';
+	position: absolute;
+	border-radius: ${() => (isV5() ? '3px' : '0')};
+`;
+
+export const GroupIcon = styled.div<{$color?: string, $variant?: 'light' | 'dark' }>`
+	${IconSize}
+	padding: 0 0 px;
+	display: inline-flex;
+	align-items: center;
+	justify-content: center;
+	border-radius: ${() => (isV5() ? '3px' : '0')};
+	
+	${({ $variant }) => ($variant === 'light' ? css`
+		color: ${({ theme }) => (isV5() ? theme.palette.base.main : '#6B778C')};
+		border: 1px solid  ${({ theme }) => (isV5() ? theme.palette.base.mid : '#E0E5F0')}; 
+	` : css`
+		color: ${({ theme }) => (isV5() ? theme.palette.primary.contrast : '#fff')}; 
+	`)};
+
+	& svg {
+		z-index: 2;
+	}
+
+	&::after {
+		background-color: ${({ $color }) => $color};
+		${PseudoElement}
+		${({ $variant }) => ($variant === 'light' ? css`
+		border: 1px solid  ${({ theme }) => (isV5() ? theme.palette.base.mid : '#E0E5F0')}; 
```

Comment:
![image](https://user-images.githubusercontent.com/62481250/179229160-37ce9693-f44e-4f82-9778-d676154a564e.png)
The bordered squares end up wider and taller. I think you should work some `box-sizing: border-box;` magic on it